src/frontend/components/UI/Sidebar/components/SidebarLinks/index.tsx

```diff
@@ -82,6 +83,22 @@ export default observer(function SidebarLinks() {
             <Images.Controller fill={sidebarSvgUnselectedFill} />
           </NavLink>
         </div>
+        {AchievementState.showAchievements && (
+          <div className="sidebarLinkGradientWrapper">
+            <NavLink
+              className={({ isActive }) =>
+                classNames('Sidebar__item', {
+                  active: isActive || location.pathname.includes('achievements')
+                })
+              }
+              end
+              to={'/achievements'}
+              onClick={async () => handleRefresh()}
```

Comment:
Async not utlized here, because is being called directly. We can use `onClick={handleRefresh}` as well

Reply:
okay updated